### ISSUE TITLE
318 afterDownloadFilesNumber: 7 

### ISSUE THREAD

**Issue by Vladimir-andyk** [ritds/figma_backup#3]
1796 afterDownloadFilesNumber: 6
1797 afterDownloadFilesNumber: 6
1798 afterDownloadFilesNumber: 6
1799 afterDownloadFilesNumber: 6
File is not downloaded during timeout

Starting to process new file (10/84), key: TurjphgpqsZaqXnUy3DkEr
Opening an empty page
В чем может быть причина?
Как утечка памяти идет

**Comment by Vladimir-andyk** [comment]
Поднял озу до 16гб и файл подкачки до 4гб
1795 afterDownloadFilesNumber: 4
1796 afterDownloadFilesNumber: 4
1797 afterDownloadFilesNumber: 4
1798 afterDownloadFilesNumber: 4
1799 afterDownloadFilesNumber: 4
File is not downloaded during timeout

Starting to process new file (36/84), key: GPE7pdMopCS8ufxG6HOwWl
Opening an empty page
TimeoutError: Navigation timeout of 180000 ms exceeded

**Comment by Vladimir-andyk** [comment]
Так, по не понятным причинам прошел дальше потом опять зависло,(переполнение памяти как 1 раз не было) думаю тайминги апи фигмы.

**Comment by Vladimir-andyk** [comment]
После обновления
waiting files to download for 19 min 0 sec.
waiting files to download for 19 min 30 sec.
waiting files to download for 20 min 0 sec.
waiting files to download for 20 min 30 sec.
waiting files to download for 21 min 0 sec.
waiting files to download for 21 min 30 sec.
waiting files to download for 22 min 0 sec.
waiting files to download for 22 min 30 sec.
waiting files to download for 23 min 0 sec.
waiting files to download for 23 min 30 sec.
waiting files to download for 24 min 0 sec.
waiting files to download for 24 min 30 sec.
waiting files to download for 25 min 0 sec.
waiting files to download for 25 min 30 sec.
waiting files to download for 26 min 0 sec.
waiting files to download for 26 min 30 sec.
waiting files to download for 27 min 0 sec.
waiting files to download for 27 min 30 sec.
waiting files to download for 28 min 0 sec.
waiting files to download for 28 min 30 sec.
waiting files to download for 29 min 0 sec.
waiting files to download for 29 min 30 sec.
waiting files to download for 30 min 0 sec.
Затиыкается на 1й минуте. На прошлой версии качалось нормально

**Comment by Vladimir-andyk** [comment]
waiting files to download for 37 min 0 sec.
waiting files to download for 37 min 30 sec.
waiting files to download for 38 min 0 sec.
waiting files to download for 38 min 30 sec.
waiting files to download for 39 min 0 sec.
В настройках 10 мин ubuntu 20.04

**Comment by masta-man** [comment]
У меня данная проблема из-за проектов с одинаковым именем.
